tests/test_basic_functionality.py

```diff
@@ -514,28 +515,47 @@ def test_to_dict_yaml(test_spec: Path) -> None:
         ("fake_key", "some_default", "some_default"),
     ],
 )
-def test_getter_single_values(
-    key: str, default_value: str, expected_value: str, getter_dict: dict[str, Any]
+def test_get_single_values(
+    key: str, default_value: str, expected_value: str, nested_dict: dict[str, Any]
 ) -> None:
-    lct = LayeredConfigTree(getter_dict)
+    lct = LayeredConfigTree(nested_dict)
 
     if default_value is None:
         assert lct.get(key) == expected_value
     else:
         assert lct.get(key, default_value) == expected_value
 
 
-def test_getter_chained(getter_dict: dict[str, Any]) -> None:
-    lct = LayeredConfigTree(getter_dict)
+def test_get_chained(nested_dict: dict[str, Any]) -> None:
+    lct = LayeredConfigTree(nested_dict)
 
-    outer_layer_2 = lct.get("outer_layer_2")
-    assert isinstance(outer_layer_2, LayeredConfigTree)
-    assert outer_layer_2.to_dict() == getter_dict["outer_layer_2"]
-    assert outer_layer_2.get("inner_layer") == "test_value2"
+    # Check that returning a LayeredConfigTree works as expected
+    assert (
+        lct.get("outer_layer_3").get("inner_layer_1").to_dict()
+        == lct.get(["outer_layer_3", "inner_layer_1"]).to_dict()
+        == nested_dict["outer_layer_3"]["inner_layer_1"]
+    )
+
+    # Check that we can get a value from a nested dict
+    assert (
+        lct.get("outer_layer_3").get("inner_layer_1").get("inner_layer_2")
+        == lct.get(["outer_layer_3", "inner_layer_1", "inner_layer_2"])
+        == nested_dict["outer_layer_3"]["inner_layer_1"]["inner_layer_2"]
+    )
 
+    # Check that the default works
+    lct.get(["outer_layer_3", "missing_key"], "foo") == "foo"
 
-def test_getter_default_values(getter_dict: dict[str, Any]) -> None:
-    lct = LayeredConfigTree(getter_dict)
+    # Check that the default only works for the last key
+    with pytest.raises(
+        ConfigurationKeyError,
+        match=re.escape("No value at key mapping '['outer_layer_3', 'whoops']'."),
+    ):
+        lct.get(["outer_layer_3", "whoops", "missing_key"], "foo")
+
+
+def test_get_default_values(nested_dict: dict[str, Any]) -> None:
```

Comment:
Not related to your work, but I don't understand the point of this test. It seems like the default value functionality was already tested in `test_get_single_value()`, and the last line of the test is just testing that the `.get()` method of `dict` works.